### PR TITLE
[AKS] BREAKING CHANGE: `az aks trustedaccess rolebinding create`: Remove deprecated `-r` options

### PR DESCRIPTION
**Pull request by YitongFeng-git**
**Related command**
az aks trustedaccess rolebinding create

**Description**

 '-r' options used to marked deprecated, now totally remove it

**Testing Guide**

**History Notes**

[AKS] BREAKING CHANGE: `az aks trustedaccess rolebinding create`: Remove deprecated '-r' options

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/acs/_params.py [hunk 1]
```diff
@@ -853,7 +853,6 @@ def load_arguments(self, _):
             'source_resource_id',
             options_list=[
                 '--source-resource-id',
-                c.deprecate(target='-r', redirect='--source-resource-id', hide=True),
             ],
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/_params.py:856]: As there's no other option form, and the default option is `--source-resource-id`, may omit this

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/azure-cli/azure/cli/command_modules/acs/_params.py (hunk 1 above)
Comment (YitongFeng-git) [reply src/azure-cli/azure/cli/command_modules/acs/_params.py]: omitted

**Review by FumingZhang — APPROVED** [review]

**Review by karataliu — APPROVED** [review]
LGTM

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**